app/src/main/java/ch/epfl/culturequest/database/FireDatabase.java

```diff
@@ -0,0 +1,32 @@
+package ch.epfl.culturequest.database;
+
+import com.google.firebase.database.FirebaseDatabase;
+
+import java.util.concurrent.CompletableFuture;
+
+public class FireDatabase implements DatabaseInterface {
```

Comment:
The class name might be ambiguous with `com.google.firebase.database.FirebaseDatabase`

Reply:
That was the name given in the bootcamp and I don't know how else we could name it 😕